simulator/Makefile

```diff
@@ -19,6 +19,14 @@ RULESPATH = $(CHIBIOS)/os/common/startup/SIMIA32/compilers/GCC
 
 PROJECT_CPU=simulator
 
+# Board-specific configuration
+ifeq (,$(filter clean,$(MAKECMDGOALS)))
+ifneq ($(BOARD_DIR),)
+  include ../firmware/$(BOARD_DIR)/board.mk
+  BOARDCPPSRC += $(BOARDS_DIR)/board_id.cpp
```

Comment:
how is board_id.cpp related to all this?

Reply:
I thought it might be important because the firmware makefile included it too. 🤷‍♂️